src/main/java/network/brightspots/rcv/Main.java

```diff
@@ -83,4 +84,13 @@ public static void main(String[] args) {
 
     System.exit(0);
   }
+
+  public static void logSystemInfo() {
+    Logger.log(Level.INFO, String.format("Launching %s version %s...", APP_NAME, APP_VERSION));
+    Logger.log(Level.INFO, "Host system: %s version %s",
+        System.getProperty("os.name"),
+        System.getProperty("os.version"));
+    Logger.log(Level.INFO, "User: %s", System.getProperty("user.name"));
```

Comment:
I'm a little iffy on including this line, as it's getting into PII territory. If you feel strongly about it, we can keep, but I'd prefer we delete it.

Reply:
Realized that this is probably somewhat of a moot point, since the file paths to the logs include username too (on Windows at least).

Reply:
I'm not sure about this one either.  I saw the api and thought, that might be useful, but it is a different and new category of data.  @CalebKleppner @gngilbert @chughes297  do you have an opinion: should we log the current user?